packages/tinacms/src/toolkit/fields/components/color-picker/block-widget.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import {
+  ColorPreview,
+  HexInput,
+  SwatchGrid,
+  WidgetProps,
+  useHexInput,
+} from './color-input';
+
+export const BlockWidget: React.FC<WidgetProps> = ({
+  presetColors,
+  color,
+  onChange,
+  width,
+}) => {
+  const {
+    inputValue,
+    handleChange,
+    handleFocus,
+    handleBlur,
+    handleSwatchClick,
+  } = useHexInput(color, onChange);
+
+  return (
+    <div
+      className='bg-white rounded-lg shadow-lg overflow-hidden'
+      style={{ width }}
```

Comment:
```suggestion
      className=`bg-white rounded-lg shadow-lg overflow-hidden w-[${width}]`
```

Code above maybe not working. But, dynamically use teh Tailwind rather than any HTML attributes.

Reply:
disregard @JackDevAU said it sucks but it's mean't to be this way :'(